src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -361,6 +437,18 @@ public String toString() {
         sb.append(" cashAdvanceKey: ").append(cashAdvanceKey).append(" cashAdvancePaymentCodeName: ").append(cashAdvancePaymentCodeName);
         sb.append(" journalPayerPaymentTypeName: ").append(journalPayerPaymentTypeName);
         sb.append(" journalPayeePaymentTypeName: ").append(journalPayeePaymentTypeName);
+        sb.append(" cashAdvanceTransactionType: ").append(cashAdvanceTransactionType);
```

Comment:
this could be simplied using ToStringBuilder, there is an example of using this in AccountingXmlDocumentAccountingLine and AmazonKfsAccountDTO

Reply:
Per our discussion, leaving code as is for now.